app/controllers/charges_controller.rb

```diff
@@ -19,6 +15,15 @@ def create
       description: 'Payment for Order'
       currency: 'usd'
     )
+binding.pry
```

Comment:
You want to avoid leaving breakpoints in your code when you push up. This will often cause the build on CI to fail or delay other builds because the previous build would still be in wait mode

Reply:
Noted 👍